pandas/core/frame.py

```diff
@@ -2156,9 +2156,10 @@ def to_parquet(
 
             .. versionadded:: 0.24.0
 
-        partition_cols : list, optional, default None
-            Column names by which to partition the dataset
-            Columns are partitioned in the order they are given
+        partition_cols : list or string, optional, default None
+            Column names by which to partition the dataset.
+            Columns are partitioned in the order they are given.
+            String identifies a single column to be partitioned.
 
             .. versionadded:: 0.24.0
```

Comment:
Can you add a `.. versionchanged:: 1.0.0` explaining that passing a single string was added in 1.0?

Reply:
I think I added it correctly. This is my first contribution so bear with me please :)

Reply:
You can put the `.. versionchanged:: ...` right below the parameter explanation (so since there is already a `..versionadded:: 0.24.0`, it would go just below that